docs/modules/ROOT/pages/how-to/measure-log-ingestion-by-organization.adoc

```diff
@@ -0,0 +1,33 @@
+= Measure Log Ingestion by Organization
+
+This guide describes how to measure log ingestion by organization.
+
+NOTE: In xref:appuio-cloud:ROOT:explanation/decisions/efk-openshift-logging.adoc[RedHat OpenShift Logging (EFK) Stack as Logging Solution] we decided to offer Elasticsearch on a fair-use policy.
+This guide helps finding organizations with higher than expected log volumes.
+
+== Prerequisites
+
+* Administrator access to the {zone}
+
+== Measure Log Ingestion by Organization
+
+. Navigate to **Observe** -> **Metrics** in the sidebar of the OpenShift console.
+
+. Use the following query to measure log ingestion by organization.
++
+[source]
+----
+sum by(organization, namespace, cluster_id) (
+  label_replace(
+    (
+      sum (
+        label_replace(
+          increase(log_collected_bytes_total{job="fluentd"}[24h]),
+          "namespace", "$1", "exported_namespace", "(.*)"
+        )
+      ) by (namespace)
+    ) * on (namespace) group_right() kube_namespace_labels{label_appuio_io_organization!=""},
+    "organization", "$1", "label_appuio_io_organization", "(.*)"
+  )
+)
```

Comment:
```suggestion
sum by(organization, namespace, cluster_id) (
  label_replace(
    increase(log_collected_bytes_total[24h]),
    "namespace",
    "$1",
    "exported_namespace",
    "(.*)"
  )
  *
  on (namespace)
  group_left(organization)
  label_replace(
    kube_namespace_labels{label_appuio_io_organization!=""},
    "organization",
    "$1",
    "label_appuio_io_organization",
    "(.+)"
  )
)
```

My take on the query. What puzzles me, ist the fact that you used `group_right` whil I have used `group_left`

Reply:
The differences are nullified by the final sum(). My query already sums up in the `group_right` step the sum just removes the superfluous labels. In your query the final sum sums up the values.

In hindsight i think your query is more clear since `group_left` is used more IMO. I'll update it.